.changeset/yellow-weeks-kick.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This changeset should be a patch bump because we follow thesemantic versioning (semver.org) specification for versioning. 

Here's a basic guide:

- Big fixes should always be limited to patches
- Minor bumps should are for features
- Major bumps are for breaking changes

Since this PR introduces a bug fix, based on semver, to a patch bump. 🙂

Reply:
Oh I see thanks for the info!

So, if I run changeset and add the changed packages but not list them under major or minor it will list it as patch. That is good to know 😄

Reply:
Yes, changesets falls back to patch if the `major` and `minor` CLI options are not selected